packages/webapp/src/components/AutoRedirect.tsx

```diff
@@ -0,0 +1,26 @@
+import type { FC } from 'react';
```

Comment:
Not sure why you added this instead of using Root, it's a bit less meaningful. Also you didn't deleted Root and you forgot to keep /getting-started redirect if dev